### DISCUSSION TITLE
how much do you use quasar

### DISCUSSION THREAD

**Discussion by Panizgeraee** [quasarframework/quasar#17617] [Polls]
just chose to do this for these reasons : data analysis , curiosity🧐 . hopefully curiosity wont kill the cat 🧐 🐈

**Comment by true-vue** [comment]
I have some doubts whether this question can give significant results. Maybe some additional context is required? Something like:

What percentage of projects you work with is using quasar.js?

Or maybe this question is targeted for quasar team members or maintainers?

**Reply by Panizgeraee** [reply]
its mainly for team members! im trying to see if they use their own project if im being honest.  :)
